Feature/WBNR-3-strategy

LourdesBl opened the pull request.

LourdesBl committed:
feature/WBNR-3-strategy

bustardcelly commented on src/pages/JoinWebinarPage/JoinWebinarPage.tsx:
```diff
@@ -178,7 +175,7 @@ const JoinWebinarPage = () => {
         {!loading && currentSection === Section.WatchParty && (
           <MainStageWithChatBox>
-            <WebinarMainStage />
+            <MainStageWrapper isWebinarStage />
```
Would this ever need to be wrapped in a `Suspense`? (I was unaware of React's capability to lazy load...)

bustardcelly committed:
Pointing conference-api
Also pointing local dev environment to latest server deploy of `wat-237-sm.red5pro.net`
